### PR TITLE
{vis}[foss/2019a] OpenSlide v3.4.1

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)
It has an extra suffix to indicate the inclusion of an as yet unmerged patch to support larger files.

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenSlide/276.diff [hunk 1]
```diff
@@ -0,0 +1,213 @@
+diff --git a/src/openslide-decode-tifflike.c b/src/openslide-decode-tifflike.c
```
Comment (Micket) [inline easybuild/easyconfigs/o/OpenSlide/276.diff:1]: Would be good to have the description and link in the patch itself instead.

I'm also not sure crazy about the name. What about `OpenSlide-3.4.1_large_file_support.patch`

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/b2a0919742427fc1dbf1c0860a098327 for a full test report.

File: easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-foss-2019a-largefiles.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+name = 'OpenSlide'
+version = '3.4.1'
+versionsuffix = '-largefiles'
+
+homepage = 'https://openslide.org/'
+description = """OpenSlide is a C library that provides a simple interface to
+read whole-slide images (also known as virtual slides)."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (Micket) [inline easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-foss-2019a-largefiles.eb:11]: I missed this until looking at vips, and noticing the chat that another person had put it at GCCcore level. Why was this moved into foss? GCC level at most i could maybe understand.

File: easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-GCCcore-8.2.0-largefiles.eb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+github_account = 'openslide'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_large_file_support.patch']
+checksums = [
+    'a5d869916e370125421535dcce778b2ba625dc50d920aa4ca93bbaaa6a7b470c',  # v3.4.1.tar.gz
+    'cb618053f4ae6c3ce37d1b8b0e4ef7c55fd17378776d13be4aa4efab91706b8c',  # OpenSlide-3.4.1_large_file_support.patch
+]
+
+# See https://github.com/openslide/openslide/pull/276 - adds large file support
```
Comment (Micket) [inline easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-GCCcore-8.2.0-largefiles.eb:22]: misplaced/leftover comment, can be dropped now that it's included in the patch itself i think.

File: easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-GCCcore-8.2.0-largefiles.eb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+builddependencies = [
```
Comment (Micket) [inline easybuild/easyconfigs/o/OpenSlide/OpenSlide-3.4.1-GCCcore-8.2.0-largefiles.eb:23]: Missing binutils builddep

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/7f42cb32ac1dfb3e90ec0e858790aa9c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
generoso-2 - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0c9b2f856d538c535ea9cf3f2c1762c5 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/67819b84ffef575a04fc4d4e53457964 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/bab78ab845d6277200039c8f8152230b for a full test report.

**Comment by lexming** [comment]
Going in, thanks @verdurin !
